docs/v1/setup-for-ubuntu-linux.sh

```diff
@@ -0,0 +1,128 @@
+#! /bin/bash
```

Comment:
We definitely do NOT need to add a script that install Madoko.  Please remove this, unless there is some reason you know of to have it here.